### PR TITLE
Adding a source icon on the Set Destination page.

### LINKED ISSUE #837: UI Polish - Set Destination
User needs some help
- could be arrows on the origin page
- https://www.google.com/search?q=community+butthole+flag&espv=2&biw=1120&bih=951&source=lnms&tbm=isch&sa=X&ved=0CAYQ_AUoAWoVChMI4MiBka-dyQIVxhk-Ch1QigQG 
- could be FTU overlay +1 [requires localization]
- could be help text in element editor next to set destination button [requires localization]

### PR DESCRIPTION
**Pull request by vazquez**
Optimized SVG.

Fixes #837

### CODE REVIEW

**Comment by vazquez** [comment]
screenshot_20151123-105030

File: src/pages/project/project.less [hunk 1]
```diff
@@ -74,6 +74,13 @@
 
     &.source {
       opacity: 0.75;
+      .shim .indicator {
+        width: 50%;
+        height: 160/220 * 50%;
```
Comment (alanmoo) [inline src/pages/project/project.less]: Where are these numbers from?

Comment (vazquez) [inline src/pages/project/project.less]: They are the same as the ones that appear on `.target .shim .indicator`. They create square bounding box based off 160:220 aspect ratio.

Comment (alanmoo) [inline src/pages/project/project.less]: Can you move them up into a variable at the top of the file? If we're going to be doing inline calculations, we should be able to trace why we're doing them.

**Comment by alanmoo** [comment]
👍

**Comment by vazquez** [comment]
:cow: :star:

**Pull request merged**